pkg/health/condition/check_backup_ready.go

```diff
@@ -74,34 +75,40 @@ func (a *backupReadyCheck) Check(ctx context.Context, etcd druidv1alpha1.Etcd) R
 	fullLeaseRenewTime := fullSnapLease.Spec.RenewTime
 	fullLeaseCreateTime := &fullSnapLease.ObjectMeta.CreationTimestamp
 
+	fullSnapshotDuration, err := utils.ComputeScheduleDuration(*etcd.Spec.Backup.FullSnapshotSchedule)
```

Comment:
What happens if `etcd.Spec.Backup.FullSnapshotSchedule` is not configured? I noticed that in `etcd-backup-restore`, it defaults to [`DefaultFullSnapshotSchedule = "0 */1 * * *"`](https://github.com/gardener/etcd-backup-restore/blob/master/pkg/types/snapshotter.go#L46).

If it not configured, then we will end up in return result as `unknow` state 

The tests are panicking because `etcd.Spec.Backup.FullSnapshotSchedule` is not set."

Reply:
Yes, forgot to adapt that. Thanks for pointing it out @seshachalam-yv . I've pushed the change now. PTAL